### PR TITLE
Autogenerated doc files as per LaPy method

### PR DESCRIPTION
**Pull request by engrosamaali91**
- The pull request is related to  whippersnappy sphinx based documentation.

- The documentation is based on LaPy project documentation i.e doc file contains autogenerated templates.

- The file config_app.py which contains a class ConfigWindow is raising stub files are missing with this method. (it could be resolved with the method I followed earlier in brain print project documentation but it  is different than the method we followed in LaPy documentation.

- Apart from stab file missing warnings only two more warning are related to indentation for _**core**_ and _read_geometry_ modules with a successful build.

### CODE REVIEW

**Comment by engrosamaali91** [comment]
In commit [a47d553](https://github.com/Deep-MI/WhipperSnapPy/pull/23/commits/a47d5533a067b02a5f37c03f3402c30c1800bfb5) the sphnix gallery has been commented out in conf.py file.

**Comment by engrosamaali91** [comment]
In commit [b14a35b](https://github.com/Deep-MI/WhipperSnapPy/pull/23/commits/b14a35b309a4cac732519c2b7b8b6d0ddbfe0116) colons have been removed from the headings and subheadings on page.

File: doc/references.bib [hunk 1]
```diff
@@ -0,0 +1,34 @@
+@article{conformal_parameterization_2020,
+ author = {Choi, Gary P. T. and Leung-Liu, Yusan and Gu, Xianfeng and Lui, Lok Ming},
+ doi = {10.1137/19M125337X},
+ journal = {SIAM Journal on Imaging Sciences},
+ number = {3},
+ pages = {1049-1083},
+ title = {Parallelizable Global Conformal Parameterization of Simply-Connected Surfaces via Partial Welding},
+ volume = {13},
+ year = {2020}
+}
```
Comment (m-reuter) [inline doc/references.bib:10]: Also here this reference should be removed. Also check below if we even use scipy in this project (I think not), then that reference can also be removed.

**Comment by m-reuter** [comment]
similarly to brainprint, it would be nice if the README.md could be imported directly, to avoid having to edit both files.

**Comment by m-reuter** [comment]
Also isort fails, please run `isort --check-only --diff .` to check what needs changing.

**Comment by engrosamaali91** [comment]
> Also isort fails, please run `isort --check-only --diff .` to check what needs changing.


conf.py file sorted. d86743ed85f28d2de6eff34e2d05ccefaeef243c

**Pull request merged**